docs/content/mixin-dev-guide/distribution.md

```diff
@@ -64,7 +64,7 @@ When `--version` is not specified, it is defaulted to `latest` which should
 represent the most recent version of the mixin.
 
 You may also choose to publish `canary` versions of the mixin, which are
-unpublished builds from the master branch. Porter and the official DeisLabs
+unpublished builds from the master branch. Porter and the official Porter Authors
```

Comment:
Seems like this can be just `the official Porter mixins`

Reply:
Updated.